Makefile

```diff
@@ -148,7 +145,6 @@ endif
 
 ALL_GRPC_BACKENDS=backend-assets/grpc/langchain-huggingface
 ALL_GRPC_BACKENDS+=backend-assets/grpc/bert-embeddings
-ALL_GRPC_BACKENDS+=backend-assets/grpc/llama
```

Comment:
as we are dropping the old backend here, we should also delete it from https://github.com/mudler/LocalAI/blob/8967ed1601b1f73a17d0255768024ccb6de709ce/pkg/model/initializers.go#L23